### ISSUE TITLE
Memorable date NVDA bug

### ISSUE THREAD

**Issue by powellkerry** [department-of-veterans-affairs/vets-design-system-documentation#1983]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Screen reader does not read validation messages for individual fields

## What I expected to happen

When you enter text for the month instead of a number, there should be a validation message specifically for the month

## Reproducing
Steps to reproduce:
1. Activate screen reader
2. tab to memorable date
3. enter Jan, or January for the month and tab to next field
4. add in valid date and year
5. after validation, the error does not mention the month and focus is not brought to the month field

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [x ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

### Details
-- Split from original issue: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1593
